web/static/css/_site-navigation.scss

```diff
@@ -1,27 +1,63 @@
-.site-navigation {
-  li {
-    display: inline-block;
+$_nav-breakpoint: 800px;
 
-    &:not(:last-child) {
-      margin-right: $small-spacing;
-    }
+.navigation-list {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: $large-spacing;
+}
+
+.navigation-logo-item {
+  @include padding($base-spacing null $small-spacing);
+  margin-right: auto;
+  width: 100%;
+
+  @include media($_nav-breakpoint) {
+    padding: 0;
+    width: auto;
   }
+}
+
+.navigation-item {
+  margin-right: $small-spacing;
+
+  @include media($_nav-breakpoint) {
+    @include margin(null 0 null $base-spacing);
+  }
+}
 
-  a {
-    color: $lighter-font-color;
-    display: inline-block;
-    font-size: $small-font-size;
-    font-weight: 600;
+.navigation-link {
+  color: $light-font-color;
+  display: block;
+  font-size: $small-font-size;
+  font-weight: 600;
```

Comment:
Do you have font weights declared in variables? Is it worth it?

Reply:
I don't think it's worth it. They're descriptive, and I've found it rare that they need to change if/when the typeface changes.